tests/test_simple.py

```diff
@@ -2,12 +2,12 @@
 
 # pylint: disable=line-too-long
 
+import dotstrings
 import os
 import sys
 import unittest
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..")))
-import dotstrings
```

Comment:
This needs to be below the `sys.path.insert` or it won't work if you haven't "installed" the package.